Contents/Code/PAsiteList.py

```diff
@@ -1588,6 +1588,7 @@
     ('^hart ', 'Hegre '),
     ('^hcm ', 'HotCrazyMess '),
     ('^hegre-art ', 'Hegre '),
+    ('^hlaf ', 'Hot Legs and Feet ')
```

Comment:
the "," is missing after the ")"

this causes the agent not to load

Reply:
Fixed